### DISCUSSION TITLE
1.51 webui run cpu mode?

### DISCUSSION THREAD

**Discussion by pinea00** [lshqqytiger/stable-diffusion-webui-directml#244] [General]
I installed the latest version of webUI 1.51 and the result is that cuda torch is automatically downloaded. It can only be executed in CPU mode?? When user-webui.bat is executed, NVIDIA driver was found??  How to let the party execute the GPU mode (I didn't change any settings) ~~thanks.

**Comment by lshqqytiger** [comment]
We detect NVIDIA driver by finding `nvidia-smi.exe` which is a part of it.
If you have installed NVIDIA driver on your PC, you may have it.
You can specify which torch backend will be used by adding `--backend ...` in commandline arguments. (like `--backend directml`)

**Reply by pinea00** [reply]
it works~thank you
